corehq/apps/app_manager/feature_support.py

```diff
@@ -14,6 +14,8 @@ def _require_minimum_version(self, minimum_version):
             return False
         assert isinstance(self.build_version, Version)
         assert isinstance(minimum_version, six.string_types + (Version,))
+        if isinstance(minimum_version, six.text_type):
+            minimum_version = minimum_version.encode('utf-8')
```

Comment:
Should this be checking `six.string_types` instead, or is the behavior meant to be only for unicode?  Feels kinda sketchy

Reply:
I think `distutils.version` can handle coercing things to versions, and it feels safer to use that then hack something together like this.  That is, I think you could instead do:
```python
if isinstance(minimum_version, six.string_types):
    minimum_version = LooseVersion(minimum_version)
```
so you're always comparing versions to versions

Reply:
This does sound like a better solution

Reply:
https://github.com/dimagi/commcare-hq/pull/20070